src/test/java/org/folio/rest/support/kafka/FakeKafkaConsumer.java

```diff
@@ -25,20 +24,31 @@
 public final class FakeKafkaConsumer {
   private final static Map<String, List<KafkaMessage<JsonObject>>> itemEvents = new ConcurrentHashMap<>();
   private final static Map<String, List<KafkaMessage<JsonObject>>> instanceEvents = new ConcurrentHashMap<>();
+  private final static Map<String, List<KafkaMessage<JsonObject>>> holdingsEvents = new ConcurrentHashMap<>();
 
   public final FakeKafkaConsumer consume(Vertx vertx) {
     final KafkaConsumer<String, String> consumer = create(vertx, consumerProperties());
 
-    consumer.subscribe(Set.of("inventory.instance", "inventory.item"));
+    consumer.subscribe(Set.of("inventory.instance", "inventory.item", "inventory.holdings-record"));
```

Comment:
Consider to use the `enum KafkaTopic getTopicName()` instead of hard-coding.

Reply:
Replaced with the enum constants.